common/test/java/com/couchbase/lite/BaseCollectionTest.kt

```diff
@@ -78,6 +78,15 @@ open class BaseCollectionTest : BaseDbTest() {
         assertEquals(n.toLong(), testCollection.count)
     }
 
+    protected fun createDocsInCustomizedCollection(n: Int, col: Collection) {
+        for (i in 0 until n) {
+            val doc = MutableDocument(String.format(Locale.US, "%s_%s_doc_%03d", col.database.name, col.name, i))
```

Comment:
There is no need to do this, in Kotlin.  I think you probably copy/pasted it from somewhere.  In Kotlin, all you need is: "${col.database.name}_${col.name}_${i}".  Just be sure you do that everywhere!